test/13_arraycommon.jl

```diff
@@ -1642,4 +1642,18 @@ end
     @test levelcode.(x) isa Vector{Union{Missing,Int16}}
 end
 
+@testset "fill()" begin
+    for ordered in (false, true), dims in ([1], [2, 3], [(2, 3)], [])
```

Comment:
just to be sure we handle all the cases correctly:
```suggestion
    for ordered in (false, true), dims in ([1], [(1,)], [2, 3], [(2, 3)], [], [()])
```